src/main/java/com/linkedin/kmf/services/ProduceService.java

```diff
@@ -141,7 +141,7 @@ private void initializeProducer() throws Exception {
     producerProps.putAll(_producerPropsOverride);
 
     _producer = (KMBaseProducer) Class.forName(_producerClassName).getConstructor(Properties.class).newInstance(producerProps);
-    LOG.info("Producer is initialized.");
+    LOG.info(_name + ": Producer is initialized.");
```

Comment:
Also not catching this exception kills the topic management service in some of our clusters so I would like to have this change merged soon.